jekyll/_cci2/executor-types.md

```diff
@@ -104,7 +109,8 @@ set the [`machine` key]({{ site.baseurl }}/2.0/configuration-reference/#machine)
 version: 2
 jobs:
   build:
-    machine: true
+    machine:
+      image: ubuntu-1604:201903-01    # pins image to specific version
 ```
 
 The default image for the machine executor is `circleci/classic:latest`.
```

Comment:
could we say something along the lines of this here?
> if you don't specify an image, you'll be given the default image - which is currently `circleci/classic:201710-01` but may change in future

Reply:
```suggestion
The default image for the machine executor is `circleci/classic:latest`.  If you don't specify an image, jobs will run on the default image - which is currently circleci/classic:201710-01 but may change in future.


```